File: app/src/main/java/com/poppin/movies/retrofit/MoviesService.java
package com.poppin.movies.retrofit;

import com.poppin.movies.models.MovieDetail;
import com.poppin.movies.models.RetornoBusca;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface MoviesService {

    @GET("/")
    Call<RetornoBusca> searchMovie(@Query("apikey") String apiKey,
                                   @Query("s") String name,
                                   @Query("type") String type);

    @GET("/")
    Call<MovieDetail> getMovie(@Query("apikey") String apiKey,
                               @Query("i") String imdbId);
}

File: app/src/main/java/com/rodolfo/movies/retrofit/MoviesService.kt
package com.rodolfo.movies.retrofit

import com.rodolfo.movies.models.MovieDetail
import com.rodolfo.movies.models.RetornoBusca

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface MoviesService {

    @GET("/")
    fun searchMovie(@Query("apikey") apiKey: String,
                    @Query("s") name: String,
                    @Query("type") type: String): Call<RetornoBusca>

    @GET("/")
    fun getMovie(@Query("apikey") apiKey: String,
                 @Query("i") imdbId: String): Call<MovieDetail>
}
